saplings/circuits/src/components/MainHeader.scss

```diff
@@ -15,15 +15,37 @@
  */
 
 .main-header {
-  align-items: center;
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
```

Comment:
This change also affected the main page. Was it intentional? 
Before: 
<img width="400" alt="Screen Shot 2020-06-18 at 7 41 31 AM" src="https://user-images.githubusercontent.com/14094978/85021242-33e22e00-b137-11ea-9a8f-de30018741fe.png">

After:
<img width="400" alt="Screen Shot 2020-06-18 at 7 39 54 AM" src="https://user-images.githubusercontent.com/14094978/85021098-f9789100-b136-11ea-9c34-d2a187ff667d.png">